manifests/b/BoxHero/BoxHero/2.2.9/BoxHero.BoxHero.locale.en-US.yaml

```
@@ -1,23 +1,24 @@
-# Created with WinGet Automation using Komac v1.11.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
-
-PackageIdentifier: BoxHero.BoxHero
-PackageVersion: 2.2.9
-PackageLocale: en-US
-Publisher: BoxHero, Inc.
-PublisherUrl: https://github.com/bgpworks
-PublisherSupportUrl: https://github.com/bgpworks/boxhero-electron/issues
-Author: bgpworks
-PackageName: BoxHero
-PackageUrl: https://github.com/bgpworks/boxhero-electron
-License: Copyright © 2020 BoxHero, Inc.
-ShortDescription: Simple Inventory Management Solution
-Tags:
-- electron
-- inventory
-ReleaseNotes: |-
-  What's Changed
-  - 안정적인 서비스 제공을 위해 데스크탑 앱을 새로 정비하였습니다.
-ReleaseNotesUrl: https://github.com/bgpworks/boxhero-electron/releases/tag/2.2.9
-ManifestType: defaultLocale
-ManifestVersion: 1.5.0
+# Created using WinGet Automation (CLI)
+
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: BoxHero.BoxHero
+PackageVersion: 2.2.9
+PackageLocale: en-US
+Publisher: BoxHero, Inc.
+PublisherUrl: https://github.com/bgpworks
+PublisherSupportUrl: https://github.com/bgpworks/boxhero-electron/issues
+Author: bgpworks
+PackageName: BoxHero
+PackageUrl: https://github.com/bgpworks/boxhero-electron
+License: Copyright © 2020 BoxHero, Inc.
+ShortDescription: Simple Inventory Management Solution
+Tags:
+- electron
+- inventory
+ReleaseNotes: |-
+  What's Changed
+  - 안정적인 서비스 제공을 위해 데스크탑 앱을 새로 정비하였습니다.
```

```suggestion
  - We have reorganized the desktop app to provide stable service.
```
Internet translation - feel free to update.